test/databaseAdapter.ts

```diff
@@ -1,16 +1,13 @@
+// DO NOT MODIFY. This file is automatically generated in initDevAndTest.ts
```

Comment:
We don't want to commit this file because it has the potential for merge conflicts.
Can you remove this from your PR.
I just made PR https://github.com/payloadcms/payload/pull/8235 to ignore this so it won't happen in the future.